app/src/main/java/com/codesoom/assignment/TodoHttpHandler.java

```diff
@@ -0,0 +1,112 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.model.Path;
+import com.codesoom.assignment.model.ResponseData;
+import com.codesoom.assignment.model.Task;
+import com.codesoom.assignment.service.TodoService;
+import com.codesoom.assignment.service.TodoGetService;
+import com.codesoom.assignment.service.TodoPostService;
+import com.codesoom.assignment.service.TodoPutService;
+import com.codesoom.assignment.service.TodoDeleteService;
+import com.codesoom.assignment.util.HttpStatus;
+import com.codesoom.assignment.util.HttpMethod;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.URI;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class TodoHttpHandler implements HttpHandler {
+    private final Map<HttpMethod, TodoService> methodMappingMap = new HashMap<>();
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    public TodoHttpHandler() {
+        initMethodMapping();
+    }
+
+    private void initMethodMapping() {
+        methodMappingMap.put(HttpMethod.GET, new TodoGetService());
+        methodMappingMap.put(HttpMethod.POST, new TodoPostService());
+        methodMappingMap.put(HttpMethod.PUT, new TodoPutService());
+        methodMappingMap.put(HttpMethod.PATCH, new TodoPutService());
+        methodMappingMap.put(HttpMethod.DELETE, new TodoDeleteService());
```

Comment:
```suggestion
        methodMappingMap.put(HttpMethod.GET, new TodoGetService());
        methodMappingMap.put(HttpMethod.POST, new TodoPostService());
        methodMappingMap.put(HttpMethod.PUT, new TodoPutService());
        methodMappingMap.put(HttpMethod.PATCH, new TodoPutService());
        methodMappingMap.put(HttpMethod.DELETE, new TodoDeleteService());
        methodMappingMap.put(HttpMethod.NULL_METHOD, new TodoDummyService());
```

이렇게 `TodoDummyService`를 추가하고, 이 더미 서비스는 아무 일도 하지 않고 `HTTP_BAD_METHOD`를 응답하도록 코딩해 둔다면 `if(service != null) {` 이 필요 없어질지도 모릅니다.